src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/InputImportOrder2.java

```diff
@@ -18,20 +18,20 @@
 package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
 import java.awt.Button; // ok
 import java.awt.Frame; // ok
-import java.awt.Dialog; // violation
+import java.awt.Dialog; // violation 'Wrong order for 'java.awt.Dialog' import.'
 import java.awt.event.ActionEvent; // ok
 import static java.awt.Button.ABORT // ok
 ;
 import javax.swing.JComponent; // ok
 import javax.swing.JTable; // ok
 import java.io.File; // ok
 import static java.io.File.createTempFile; // ok
-import java.io.IOException; // violation
+import java.io.IOException; // violation 'Wrong order for 'java.io.IOException' import.'
 import java.io.InputStream; // ok
 import java.io.Reader; // ok
-import static javax.swing.WindowConstants.*; // violation
+import static javax.swing.WindowConstants.*; // violation 'Wrong order for 'javax.swing.WindowConstants.*' import.'
 
-import static sun.tools.util.ModifierFilter.ALL_ACCESS; // violation
+import static sun.tools.util.ModifierFilter.ALL_ACCESS; // violation 'Extra separation in import group before 'sun.tools.util.ModifierFilter.ALL_ACCESS''
```

Comment:
I am not sure why Line length validation is not complaining.
But let's keep messaging on limit 120 symbols
`Extra separation in import group before .*`

Please apply same for others long lines.

Reply:
Thanks for reviewing